plone/app/layout/viewlets/common.py

```diff
@@ -324,11 +326,14 @@ def navtree(self):
             'plone.types_use_view_action_in_listings', [])
 
         for brain in brains:
-            brain_path = '/'.join(brain.getPath().split('/'))
+            brain_path = brain.getPath()
```

Comment:
anybody has an idea, why that `split` and `join` combination was there? I just removed it...

Reply:
:dancer:

Reply:
LOL!